TvMaterialCatalog/gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-agp = "8.3.1"
+agp = "8.3.2"
```

Comment:
We should update AGP version to "8.4.0" as Android Studio Jellyfish became stable.